src/libpgagroal/utils.c

```diff
@@ -797,13 +817,17 @@ pgagroal_set_connection_proc_title(int argc, char** argv, struct connection* con
 {
    struct configuration* config;
    int primary;
-   char info[MISC_LENGTH];
+   char info[MAX_PROCESS_TITLE_LENGTH];
```

Comment:
We should `memset(&info, 0, sizeof(info));` this

Reply:
Shame on me, right.